pxt.json

```diff
@@ -1,14 +1,16 @@
 {
     "name": "motobit",
-    "version": "0.0.3",
+    "version": "0.0.4",
```

Comment:
@soubsoub could you change this back to 0.0.3?

Version 0.0.4 was for your local fork for testing purposes, but the main moto-bit repo is still on version 0.0.3

Reply:
This has now been fixed